mobile-app/components/Board.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import Svg from 'react-native-svg';
+import PlayableCell from './PlayableCell';
+import BorderCell from './BorderCell';
+import { GameState } from '../../web-app/src/common/gameState';
+import { CellType } from "../utils";
+
+const CELL_STROKE_COLOR = "white";
+const CELL_SIZE = 20;
+const DISTANCE_BETWEEN_HEXAGONE_PARALLEL_SIDES = Math.sqrt(3);
+
+interface BoardData {
+  gameState: GameState;
+}
+
+export default function Board(props: BoardData) {
+  return (
+    <Svg width={getApproximateSvgSize(props.gameState.board.length)} height={getApproximateSvgSize(props.gameState.board.length)}>

```

Comment:
`getApproximateSvgSize(props.gameState.board.length)` is repeated, extract it to a const outside of the return JSX